src/meta-srv/src/handler/failure_handler.rs

```diff
@@ -88,8 +88,8 @@ impl HeartbeatHandler for RegionFailureHandler {
                     table: x.table.clone(),
                     cluster_id: stat.cluster_id,
                     datanode_id: stat.id,
-                    // TODO(#1566): Use the real table id.
-                    table_id: 0,
+                    // TODO(weny): Update proto's table_id type to u32
+                    table_id: x.table_id as u32,
```

Comment:
Is this cast safe?

Reply:
According to the definition in the codebase, the type of table_id should be u32. I'm correcting this wrong definition in the proto repository. GreptimeTeam/greptime-proto#40